src/index.html

```diff
@@ -104,19 +104,24 @@ <h4 class="opacity-60">GitHub Discussions</h4>
         <div class="max-width-container">
           <div class="sm:flex sm:flex-row gap-10">
```

Comment:
This gets a little close to the hero component on mobile, so we should add a bit of space.

```suggestion
          <div class="mt-14 sm:flex sm:flex-row sm:mt-0 gap-10">
```